.github/workflows/release.yml

```
@@ -0,0 +1,81 @@
+name: Publish Module
+
+on:
+  workflow_call:
+# In the repo that uses this workflow, this should be set to
+# on: workflow_dispatch
+
+jobs:
+  publish-module:
+    name: Tag Release and Publish to Forge
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
```

I think that v1 and v2 are deprecated
```suggestion
        uses: actions/checkout@v2
```